var/spack/repos/builtin/packages/ectrans/package.py

```diff
@@ -59,6 +59,9 @@ class Ectrans(CMakePackage):
     depends_on("fiat~mpi", when="~mpi")
     depends_on("fiat+mpi", when="+mpi")
 
+    # https://github.com/ecmwf-ifs/ectrans/issues/194
+    conflicts("%oneapi@2025:", when="@1.5.0:1.5.1")
```

Comment:
Didn't you discover that the fault also occurs with version 1.4.0? If so, should this conflict spec include version 1.4.0?